doc/Gabc.tex

```diff
@@ -283,6 +282,15 @@ \subsection{Simple Slurs}
 Use \texttt{uslur} instead of \texttt{oslur} (with either form) to create an
 under-the-staff slur.
 
+\subsection{Horizontal episema placement for very high and low notes}
+
+Gregorio places horizontal episemus under c and above k closer to the notes when no ledger line is present.
```

Comment:
As discussed before, episema is Latin 3rd declension, with citation form episema, episematis, neuter gender.  So the singular nominative or accusative form would be episema and the plural form is episemata.  So here, "episemus" should be replaced with episemata.

Also, `k` is not always the pitch below the ledger line; it depends on the number of staff lines.